buildkite/scripts/caqti-upgrade-plus-archive-init-speedup.patch

```diff
@@ -2795,784 +2795,7 @@ index caa4203939..a2934a6184 100644
    | Error e ->
        [%log fatal]
          ~metadata:[ ("error", `String (Caqti_error.show e)) ]
-diff --git a/src/app/berkeley_account_tables/berkeley_account_tables.ml b/src/app/berkeley_account_tables/berkeley_account_tables.ml
```

Comment:
This patch was useful (it upgraded caqti which allowed us to optimize resource usage which was critical for replayer, but might appear to be critical for something else later)

Let's create a follow-up PR/issue to have this patched upstreamed

Reply:
I just removed diff for apps i removed. I will create follow up PR for applying this patch